src/options.rs

```diff
@@ -234,6 +234,14 @@ where
                 )
                 .value_name("prefix")
                 .takes_value(true),
+            Arg::with_name("anon-fields-prefix")
+                .long("anon-fields-prefix")
+                .help(
+                    "Use the given prefix for the anon fields instead of \
+                     __bindgen_anon_.",
+                )
+                .value_name("prefix")
+                .takes_value(true),
```

Comment:
Is it worth using `.default_value("__bindgen_anon_")` here ? that makes the output of `bindgen --help` show the default :
```
--anon-fields-prefix <prefix>          Use the given prefix for the anon fields. [default: __bindgen_anon_]
```

This could allow us to avoid repeating `__bindgen_anon_` a lot of places.

Stylistically, your existing code fits the surrounding code better, but ergonomically for the user, and for the code maintainer, it might be nice to use the `.default_value()` feature here.

There are some downsides to my suggestion too, so this is just an idea.

You can see my thoughts [in my `feature/add-anon-fields-prefix-option` branch](https://github.com/varphone/rust-bindgen/compare/feature/add-anon-fields-prefix-option...kulp:feature/add-anon-fields-prefix-option).

Reply:
Yeah, your solution is greate，thanks for the changed。